src/main/scala/zio/prelude/IdentityBoth.scala

```diff
@@ -99,6 +139,16 @@ object IdentityBoth extends LawfulF.Invariant[DeriveEqualIdentityBothInvariant,
       def both[A, B](fa: => Id[A], fb: => Id[B]): Id[(A, B)] =
         Id(Id.unwrap(fa) -> Id.unwrap(fb))
     }
+
+  /**
+   * The `IdentityBoth` instance for `Try`.
+   */
+  implicit def TryIdentityBoth[R, E]: IdentityBoth[Try] =
```

Comment:
`implicit val` here since the type parameters `R` and `E` are extraneous here.

Reply:
Yeah, I don't know why I included R and E, thanks!